Spigot-API-Patches/0282-fix-Inventory-getContents-null-annotations.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: CDFN <codefun@protonmail.com>
+Date: Fri, 12 Mar 2021 18:31:31 +0100
+Subject: [PATCH] fix Inventory#getContents null annotations
+
+
+diff --git a/src/main/java/org/bukkit/inventory/Inventory.java b/src/main/java/org/bukkit/inventory/Inventory.java
+index 6386206188e820206bb1a9f516b5e194fdc9d952..0d3572192aa67d75340fd410d6a84e7c23db320d 100644
+--- a/src/main/java/org/bukkit/inventory/Inventory.java
++++ b/src/main/java/org/bukkit/inventory/Inventory.java
+@@ -158,9 +158,8 @@ public interface Inventory extends Iterable<ItemStack> {
+      *
+      * @return An array of ItemStacks from the inventory. Individual items may be null.
+      */
+-    @NotNull
+-    public ItemStack[] getContents();
+-
++    // Paper - make array type nullable instead array
++    public @Nullable ItemStack @NotNull [] getContents();
```

Comment:
Shouldn't the `// Paper` comment be on the line of the method declaration, instead of the line above?
Personally, I would also reword the comment as follows: "make array **elements** nullable instead of array".

Reply:
yeah put it on the same line. also, dont we have another patch to fixed wrong spigot nullability annotations already? should be merged with that

Reply:
![image](https://user-images.githubusercontent.com/41289688/111051699-767e3000-8455-11eb-9a3e-8cfab9a5420d.png)
I don't think there's patch for it. From what I see in git log it's just patch per fix.